src/renderer/components/data-settings/data-settings.js

```diff
@@ -1015,6 +1019,55 @@ export default defineComponent({
       await this.promptAndWriteToFile(options, JSON.stringify(this.allPlaylists), 'All playlists has been successfully exported')
     },
 
+    exportPlaylistsForOlderVersionsSometimes: function () {
+      if (this.shouldExportPlaylistForOlderVersions) {
+        this.exportPlaylistsForOlderVersions()
+      } else {
+        this.exportPlaylists()
+      }
+    },
+
+    exportPlaylistsForOlderVersions: async function () {
+      const dateStr = getTodayDateStrLocalTimezone()
+      const exportFileName = 'freetube-playlists-for-single-favorites-playlist-' + dateStr + '.db'
+
+      const options = {
+        defaultPath: exportFileName,
+        filters: [
+          {
+            name: 'Database File',
+            extensions: ['db']
+          }
+        ]
+      }
+
+      const favoritesPlaylistData = {
+        playlistName: 'Favorites',
+        protected: true,
+        videos: [],
+      }
+
+      this.allPlaylists.forEach((playlist) => {
+        playlist.videos.forEach((video) => {
+          const videoAlreadyAdded = favoritesPlaylistData.videos.some((v) => {
+            return v.videoId === video.videoId
+          })
+          if (videoAlreadyAdded) { return }
+
+          favoritesPlaylistData.videos.push(
+            Object.assign({
+              // The "required" keys during import (but actually unused) in older versions
+              isLive: false,
+              paid: false,
+              published: '',
+            }, video)
+          )
+        })
+      })
+
+      await this.promptAndWriteToFile(options, JSON.stringify([favoritesPlaylistData]), 'All playlists has been successfully exported')
```

Comment:
**nitpick:** String needs to be localized (multiple instances)

Reply:
It's a translation key suffix
![image](https://github.com/PikachuEXE/FreeTube/assets/1018543/3f362158-e628-4ad8-9d75-70054831f742)

Reply:
Thanks, didn't see that - I forgot about this from recent linting Chunky's PR